powerauth-fido2/src/main/java/com/wultra/powerauth/fido2/rest/model/converter/AssertionChallengeConverter.java

```diff
@@ -113,7 +115,6 @@ public AssertionChallenge convertAssertionChallengeFromOperationDetail(Operation
                 final List<String> transports = (List<String>) ad.getExtras().get("transports");
                 final String aaguid = (String) ad.getExtras().get("aaguid");
 
-                // Obtain credential ID, append data to credential ID if the authenticator is a Wultra authenticator that supports visual challenge.
```

Comment:
Thanks, the comment was really outdated. If you touched the line, `credentialId` could by final.